test/e2e/flowaggregator_test.go

```diff
@@ -1904,23 +1904,14 @@ func testL7FlowExporterController(t *testing.T, data *TestData, isIPv6 bool) {
 	testFlow1 := testFlow{
 		srcPodName: clientPodName,
 	}
-	var cmd []string
 	if !isIPv6 {
 		testFlow1.srcIP = clientPodIPs.IPv4.String()
 		testFlow1.dstIP = serverIPs.IPv4.String()
-		cmd = []string{
-			"curl",
-			fmt.Sprintf("http://%s:%d", serverIPs.IPv4.String(), serverPodPort),
-		}
 	} else {
 		testFlow1.srcIP = clientPodIPs.IPv6.String()
 		testFlow1.dstIP = serverIPs.IPv6.String()
-		cmd = []string{
-			"curl",
-			"-6",
-			fmt.Sprintf("http://[%s]:%d", serverIPs.IPv6.String(), serverPodPort),
-		}
 	}
+	cmd := []string{"curl", getURL(testFlow1.dstIP, fmt.Sprint(serverPodPort))}
```

Comment:
Not sure if the port argument is always int or not, if it's always int, maybe add a new parameter to `getURL` as the second argument, and then we don't have to call the format converter like `fmt.Sprint(serverPodPort)` every time when there is a port, which can be done inside of the `getURL`.

Reply:
done